src/Microsoft.ML.KMeansClustering/KMeansModelParameters.cs

```diff
@@ -134,11 +138,11 @@ private KMeansPredictor(IHostEnvironment env, ModelLoadContext ctx)
 
             InitPredictor();
 
-            InputType = new VectorType(NumberType.Float, _dimensionality);
-            OutputType = new VectorType(NumberType.Float, _k);
+            _inputType = new VectorType(NumberType.Float, _dimensionality);
+            _outputType = new VectorType(NumberType.Float, _k);
         }
 
-        public ValueMapper<TIn, TOut> GetMapper<TIn, TOut>()
+        ValueMapper<TIn, TOut> IValueMapper.GetMapper<TIn, TOut>()
```

Comment:
>IValueMapper.GetMapper [](start = 31, length = 22)

So it's great we did this here, should we do this for all implementations now that `IValueMapper` is internal?